.editorconfig

```diff
@@ -0,0 +1,13 @@
+# top-most EditorConfig file
+root = true
+
+[*]
+# Unix-style newlines with a newline ending every file
+end_of_line = lf
+insert_final_newline = true
+
+indent_style = space
+indent_size = 4
```

Comment:
The discussion didn't really arrive at consensus on this, right?

Reply:
It did not. But this PR is a step towards one :)

Reply:
are we keeping the same for css files as well?
those look all consistent with 2 spaces.

Are there are processed files js/css files that need to be excluded?

Reply:
Aren't processed/auto-generated files irrelevant, since by definition we don't manually edit them?